frontend/stylesheets/main.less

```
@@ -20,12 +20,16 @@
 
 @font-face {
   font-family: "Nunito";
-  src: local("Nunito"), url("Nunito-VariableFont_wght.ttf") format("truetype");
+  src:
+    local("Nunito"),
+    url("Nunito-VariableFont_wght.ttf") format("truetype");
 }
 
 @font-face {
   font-family: "Nunito";
```

Stylelint (17.9.0)

[error] Expected no quotes around "Nunito" (font-family-name-quotes)

(font-family-name-quotes)